lib/segment/src/index/field_index/geo_index.rs

```diff
@@ -469,6 +469,16 @@ impl GeoMapIndex {
     pub fn values_count(&self, point_id: PointOffsetType) -> usize {
         self.get_values(point_id).map(|x| x.len()).unwrap_or(0)
     }
+
+    pub fn values_is_empty(&self, point_id: PointOffsetType) -> bool {
+        self.get_values(point_id)
+            .map(|x| x.is_empty())
+            .unwrap_or(true)
+    }
+
+    pub fn values_is_none(&self, point_id: PointOffsetType) -> bool {
+        self.get_values(point_id).is_none()
+    }
```

Comment:
This check is not accurate. Index doesn't store explicit `null` value, so `{"a": null}` and `{}` will both have same representation in the index, also condition should work differently

Reply:
Then should we mention in our docs that to prefer `is_empty` over `is_null` because of possible performance?